lib/v2/gov/jinanhealthcommission/medical_exam_notice.js

```diff
@@ -0,0 +1,38 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://jnmhc.jinan.gov.cn';
+    const url = 'https://jnmhc.jinan.gov.cn/module/web/jpage/dataproxy.jsp?page=1&webid=28&path=/&columnid=14418&unitid=18878&webname=%25E6%25B5%258E%25E5%258D%2597%25E5%25B8%2582%25E5%258D%25AB%25E7%2594%259F%25E5%2581%25A5%25E5%25BA%25B7%25E5%25A7%2594%25E5%2591%2598%25E4%25BC%259A&permissiontype=0';
```

Comment:
```suggestion
    const res = await got('https://jnmhc.jinan.gov.cn/module/web/jpage/dataproxy.jsp', {
        searchParams: {
            page: 1,
            webid: 28,
            path: '/',
            columnid: 14418,
            unitid: 18878,
            webname: '济南市卫生健康委员会',
            permissiontype: 0,
        },
    });
```

Reply:
Thank you. The suggested chage has bee made.